etc/orthography.tsv

```diff
@@ -1,472 +1,335 @@
+k	k	702	U+006B	"mɑkɑ̱t,puttsuku,tūkúgɑ"	Tetelcingo Mexicano, Kawaiisu, Orayvi Hopi, N Tepehuan, Southern Ute, Big Smokey Valley Shoshone, Comanche, Pipil, Tubar, Western Shoshone, Rarámuri, Arizona Yoeme (Yaqui), Chemehuevi, Southern Paiute, Classical Nahuatl, Tümpisa Shoshone (Panamint,Koso), Wixarika (Huichol), Cahuilla, Pochutla Mexicano, Tongva (Gabrielino), Cora, Shoshone, Pahkannil (Tübatulabal), Mayo, Northern Paiute, Zuni, Eudeve, Western Mono, Guarijío, Kitanemuk, Cupeño, S Tepehuan, Tohono O'odham, Ópata, Pima Bajo, Luiseño, Acjachemen (Juaneño), Tewa, Ipai, Serrano
+l	l	354	U+006C	"kitlɑ̄tiɑ,kwɑwitl,sītˡɑlin"	Luiseño, Tetelcingo Mexicano, Zuni, Pipil, Classical Nahuatl, Cahuilla, Rarámuri, Pahkannil (Tübatulabal), Tubar, Cupeño, Acjachemen (Juaneño), Mayo, Ipai, Arizona Yoeme (Yaqui), Guarijío, Orayvi Hopi, Pima Bajo, Pochutla Mexicano, Tohono O'odham, N Tepehuan, Cora, Wixarika (Huichol)
+m	m	502	U+006D	"kokmɯr,tɑm,tɑʳmoʳʔ"	Cahuilla, Tümpisa Shoshone (Panamint,Koso), Ipai, Rarámuri, Southern Paiute, Orayvi Hopi, Big Smokey Valley Shoshone, Tubar, Luiseño, Tetelcingo Mexicano, Tongva (Gabrielino), Shoshone, Kawaiisu, N Tepehuan, Guarijío, Kitanemuk, Comanche, Arizona Yoeme (Yaqui), Western Shoshone, Cupeño, Serrano, Chemehuevi, Pahkannil (Tübatulabal), Classical Nahuatl, Acjachemen (Juaneño), Pipil, Wixarika (Huichol), Western Mono, Northern Paiute, Pima Bajo, Pochutla Mexicano, Mayo, Southern Ute, Ópata, Eudeve, Zuni, Cora, Tewa, Tohono O'odham, S Tepehuan
+n	n	729	U+006E	"su̱niʎ,tuen,tɯtɯnɑ"	Guarijío, Tetelcingo Mexicano, Kawaiisu, Big Smokey Valley Shoshone, Comanche, Western Mono, Zuni, Pahkannil (Tübatulabal), Shoshone, Pipil, Tongva (Gabrielino), Cora, Southern Ute, Tümpisa Shoshone (Panamint,Koso), Orayvi Hopi, Chemehuevi, Classical Nahuatl, Cahuilla, Cupeño, Western Shoshone, Arizona Yoeme (Yaqui), Tubar, Rarámuri, Northern Paiute, Mayo, Pochutla Mexicano, Luiseño, Southern Paiute, Wixarika (Huichol), Eudeve, Kitanemuk, N Tepehuan, Ipai, Ópata, Acjachemen (Juaneño), S Tepehuan, Serrano, Pima Bajo, Tewa, Tohono O'odham
+o	o	706	U+006F	"monojo,mopi,sikoɖ"	Tongva (Gabrielino), Tetelcingo Mexicano, Zuni, Orayvi Hopi, Kawaiisu, Big Smokey Valley Shoshone, Guarijío, Tohono O'odham, Tümpisa Shoshone (Panamint,Koso), Tubar, Classical Nahuatl, Western Shoshone, Comanche, S Tepehuan, Pima Bajo, Chemehuevi, Northern Paiute, Shoshone, Western Mono, Kitanemuk, Rarámuri, Pochutla Mexicano, Tewa, Arizona Yoeme (Yaqui), Mayo, Acjachemen (Juaneño), Southern Ute, N Tepehuan, Luiseño, Eudeve, Pahkannil (Tübatulabal), Ópata, Southern Paiute, Serr
```

Comment:
`ŋw` as `ŋ w` may be reasonable, but what if the data is mixed, and this is in fact `ŋ_w`? YOu have the same languages in the line below, listing this as a phoneme.

Reply:
The language field is being carried over from the original profile, I was not touching it. I suppose I can collect it as well (in the form of the language id).

As for the `w` vs. `_w`, once more it is following the original data, keeping track of both the IPA and LaTeX TIPA fields. I can map all to a labialization diacritic if you want.

Reply:
I would rather like you to investigate if this is realistic, to have a distinction in the SAME language, of k_w and `k w`. If it is not, I suppose you unify it. I find it strange to see these things there, and my suspicion is that the original data was sloppily collected.

Reply:
Yes, phonemic distinction between labialized and non-labialized (whether geminated or not) is described as a common Uto-Aztecan feature, even more among Numic languages, and it is a realistic distinction in my view. I cannot find a good vocabulary online that is not in some way related to this dataset, but from the phonological description there is no issue (see, for example, [Phoible on this language](https://phoible.org/inventories/view/1133), the Wikipedia article [for the family](https://en.wikipedia.org/wiki/Numic_languages) and [for the language](https://en.wikipedia.org/wiki/Timbisha_language), or [Peiros](http://www.jolr.ru/files/(49)jlr2010-4(90-116).pdf) [sorry for some of the references...]). Campbell (1997) is also explicit about it when describing the family (pp.133--8), but the best reference is [Dayley (1989)](https://scholarworks.boisestate.edu/cgi/viewcontent.cgi?article=1360&context=fac_books) (p. 406 on in the PDF) -- I could not find a minimal pair, but the description clearly assumes the distinction is there.

Now, it is true that this might be an artefact of the underlying phonological theory/analysis, or that it might be not be distinctive from the point-of-view of cognate judgement, but it is described as distinctive and it seems to be properly mirrored in the raw data. I would keep them as they are, but let me know if you prefer them split.